vpr/src/place/noc_place_utils.h

```diff
@@ -14,28 +14,26 @@
 #include "fstream"
 
 // represent the maximum values of the NoC cost normalization factors //
-// we need to handle the case where the agggregate bandwidth is 0, so we set this to some arbritary positive number that is greater than 1.e-9, since that is the range we expect the normalization factor to be (in Gbps)
+// we need to handle the case where the aggregate bandwidth is 0, so we set this to some arbitrary positive number that is greater than 1.e-9, since that is the range we expect the normalization factor to be (in Gbps)
 constexpr double MAX_INV_NOC_AGGREGATE_BANDWIDTH_COST = 1.;
-// we expect the latency costs to be in the picosecond range, and we don't expect it to go lower than that. So if the latency costs go below the picosecond range we trim the normalization value to be no higher than 1/ps
+// we expect the latency costs to be in the pico-second range, and we don't expect it to go lower than that. So if the latency costs go below the pico-second range we trim the normalization value to be no higher than 1/ps
 // This should be updated if the delays become lower
 constexpr double MAX_INV_NOC_LATENCY_COST = 1.e12;
 
 // we don't expect the noc_latency cost to ever go below 1 pico second.
 // So this value represents the lowest possible latency cost.
 constexpr double MIN_EXPECTED_NOC_LATENCY_COST = 1.e-12;
 
-/* Defines how the links found in a traffic flow are updated in terms
- * of their bandwidth usage.
+/**
+ * @brief Each traffic flow cost consists of two components:
+ *        1) traffic flow aggregate bandwidth
```

Comment:
(sum over all used links of the traffic_flow bandwidth)

and

(currently unloaded / best-case latency of the flow)